examples/gatekeeper-imperative/README.md

```diff
@@ -15,21 +23,15 @@ We have a `ConfigMap` in `config-map.yaml` that violates the policy.
 
 ### Function invocation
 
-Get the package:
-
-```shell
-$ kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/gatekeeper/imperative
-```
-
 Run the function with `--results-dir` flag:
 
 ```shell
-$ kpt fn eval --image=gcr.io/kpt-fn/gatekeeper:unstable --results-dir=results
+$ kpt fn eval gatekeeper-imperative --image=gcr.io/kpt-fn/gatekeeper:unstable --results-dir=/tmp
 ```
 
 ### Expected result
 
-Let's take a look at the structured result:
+Let's take a look at the structured result in `/tmp/results.yaml`:
 
 ```yaml
 apiVersion: kpt.dev/v1alpha2
```

Comment:
once, we uprev the versions to `v1`, we will have to update these docs, but `find/replace` should be enough.

Reply:
Phani is taking care of it in https://github.com/GoogleContainerTools/kpt-functions-catalog/pull/408